docs/source/setup/recipes.rst

```diff
@@ -67,3 +67,222 @@ example, support for rabbitMQ message bus or web support), as well asl supportin
+
+These are all achived with an inventory that looks like:
+
+.. literalinclude:: ../../../examples/deployment/recipe-inventory.yml
+   :caption: recipe-inventory.yml
+   :language: yaml
+   :linenos:
+
+You could expand the inventory to assign values to any of the variables documented in the
+:ref:`recipes-configuration` section. These can be applied to specifc hosts, or to groups per the
+ansible inventory system.
+
+Similarly, we can modify the platform configuration by updating the platform configuration file.
+For example, we'll set collector1 to use the RabbitMQ message bus by replacing the empty dictionary
+with that configuration as seen here:
+
+.. literalinclude:: ../../../examples/deployment/collector1/collector1.yml
+   :caption: collector1/collector1.yml
+   :language: yaml
+   :linenos:
+
+Step 2: Configured the systems
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+With the configuration files written, we can start running some actual recipes. The first needed
+is host-config. Since this needs admin access on the remote systems, we add the ``-K`` flag, so
+the execution command looks like ``ansible-playbook -i <path/to/your/inventory>.yml <path/to/>volttron/deployment/recipes/host-config.yml -K``.
+Take note of the output while running, each step reports on the action taken on every remote,
+which may differ based on configuration choices made in the prior step.
+
+Step 3: Install the platform
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Having configured the system in the last step, we now are able to install and configure the
+VOLTTRON platform. Because this is a user-space process, the ``-K`` flag is not used. This
+command looks like ``ansible-playbook -i <path/to/your/inventory>.yml <path/to/>volttron/deployment/recipes/install-platform.yml``.
+Again, if you review the console output your will see that platform configuration differences
+are reflected.
+
+Having completed this step, many components have been added to the user's home directory on
+the remote systems (or whatever alternate location configured in the inventory file). If you
+ssh to those systems you can inspect those directories and files which have been created.
+
+Step 4: Start the platform
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Starting the platform follows the same pattern as the prior two steps, the command is
+``ansible-playbook -i <path/to/your/inventory>.yml <path/to/>volttron/deployment/recipes/install-platform.yml``.
+This particular playbook simply starts the platform (assuming it is not already running).
+
+If you connect to one of the remote systems, you can source the activation script created
+during the platform installation step. This activates the VOLTTRON environment as well as
+setting appropriate environment variables. If you run ``vctl``, you'll see that the platform
+is running (but currently has no agents installed).
+
+Extra steps
+~~~~~~~~~~~
+
+Having started the platform, you can leverage ansible's ad-hoc commands to interact with them.
```

Comment:
The only other thing I might add in this is the limit to specific host command line option, but doesn't have to be there.

Reply:
Added a reference here and also earlier